frontend/src/components/HandbookOverlay/HandbookSystemRole.test.tsx

```diff
@@ -121,4 +124,41 @@ describe('HandbookSystemRole component tests', () => {
 		const lockedBox = container.getElementsByClassName('role-locked');
 		expect(lockedBox).toHaveLength(0);
 	});
+
+	test('renders an error message when no system roles are passed to it', () => {
+		const numLevelsCompleted = 0;
+
+		render(
+			<HandbookSystemRole
+				numCompletedLevels={numLevelsCompleted}
+				systemRoles={emptySystemRoles}
+			/>
+		);
+
+		expect(screen.getByText('System Roles')).toBeInTheDocument();
+
+		// make sure no system roles are displayed on the page
+		expect(screen.queryAllByText(level1SystemRole)).toHaveLength(0);
+		expect(screen.queryAllByText(level2SystemRole)).toHaveLength(0);
+		expect(screen.queryAllByText(level3SystemRole)).toHaveLength(0);
+
+		// check that the error message is displayed
+		expect(screen.queryAllByText(errorMessage)).toHaveLength(1);
```

Comment:
Prefer jest-dom matchers, and prefer single-match calls when you expect none or one match:

```
expect(screen.getByText(errorMessage)).toBeInTheDocument();
```